### PR TITLE
Release 5.5.0

### PR DESCRIPTION
**Pull request by d1nuc0m**
New release request

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,32 @@
...
 All notable changes to this project will be documented in this file.
 
+## [v5.5.0](https://github.com/voxpupuli/puppetboard/tree/v5.5.0) (2024-11-24)
+
+[Full Changelog](https://github.com/voxpupuli/puppetboard/compare/v5.4.0...v5.5.0)
+
+**Features**
+
+- Add support for Python 3.12, 3.13
+- Drop support for Python 3.8
```
Comment (ekohl) [inline CHANGELOG.md:12]: Isn't this typically a backwards incompatible change?

Comment (smortex) [reply CHANGELOG.md:12]: It is, the PR (#1184) is properly tagged, my guess is that this has not been generated by github-changelog-generator.

@d1nuc0m can you check you generated with something like `bundle exec rake changelog`?

Comment (d1nuc0m) [reply CHANGELOG.md]: Nevermind, found [RELEASE.MD](https://github.com/voxpupuli/puppetboard/blob/master/RELEASE.md)

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,53 @@
 
 All notable changes to this project will be documented in this file.
 
+## [v5.5.0](https://github.com/voxpupuli/puppetboard/tree/v5.5.0) (2024-11-26)
+
+[Full Changelog](https://github.com/voxpupuli/puppetboard/compare/v5.4.0...v5.5.0)
+
+**Breaking changes:**
+
+- Drop support for python 3.8 [\#1184](https://github.com/voxpupuli/puppetboard/pull/1184) ([TheMeier](https://github.com/TheMeier))
```
Comment (bastelfreak) [inline CHANGELOG.md:11]: because of this breaking change, it needs to be a major release (6.0.0).

Comment (d1nuc0m) [reply CHANGELOG.md:11]: Ok, see #1200

**Pull request closed without merging**